api/curve/v1/api/data_dataName.py

```diff
@@ -82,15 +83,17 @@ def post(self, dataName):
             return self.render(msg='%s is exists' % data_name, status=422)
         if len(request.files) < 1:
             return self.render(msg='expect file input', status=422)
-        upload_file = request.files.values()[0]
+        upload_file = request.files['file']
+        current_app.logger.info('Info: %s %s %s', request.files['file'], request.files, request.files['file'].filename)
         try:
             points, time_formatter = self._parse_file(upload_file)  # parse data in csv
         except Exception as e:
-            return self.render(msg=e.message, status=422)
+            return self.render(msg=str(e), status=422)
         if len(points) < 2:
             return self.render(msg='at least 2 point', status=422)
         timestamps = np.asarray(sorted(points.keys()))
         periods = np.diff(timestamps)
+        #current_app.logger.info('Periods as numpy %s', points.keys())
```

Comment:
It's better to delete the personal debug code.

Reply:
I have corrected this in the new PR.